### DISCUSSION TITLE
could hfs3 only listen 127.0.0.1

### DISCUSSION THREAD

**Discussion by armgong** [rejetto/hfs#358] [General]
because our IT management rules, we need set hfs3 behind nginx (which it allowed), so only listen on 127.0.0.1 is enough for us.  
and  our IT department don't allow hfs3 listen on LAN IP (10.* or 192.*), only 127.0.0.1 allowed.
so the question is could hfs3 only listen 127.0.0.1?

**Comment by armgong** [comment]
sorry duplicated

**Comment by rejetto** [comment]
Hey, don't have it yet, but I can add this possibility easily in next release.

**Reply by armgong** [reply]
greate , thanks a lot
